src/extensions/fixed-columns/bootstrap-table-fixed-columns.js

```diff
@@ -233,6 +233,26 @@ $.BootstrapTable = class extends $.BootstrapTable {
       this.$fixedBodyRight.scrollLeft(this.$fixedBodyRight.find('table').width())
       this.$fixedBodyRight.css('overflow-y', this.options.height ? 'auto' : 'hidden')
     }
+
+    const that = this
+    if (this.options.filterControl) {
```

Comment:
It's better to move this event listener to `initFixedColumnsEvents ` function.

Reply:
done